src/renderer/components/status-bar/status-bar.test.tsx

```diff
@@ -0,0 +1,109 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+
+import React from "react";
+import "@testing-library/jest-dom/extend-expect";
+import { StatusBar } from "./status-bar";
+import { getDiForUnitTesting } from "../../getDiForUnitTesting";
+import type { DiRender } from "../test-utils/renderFor";
+import { renderFor } from "../test-utils/renderFor";
+import { computed } from "mobx";
+import type { ConfigurableDependencyInjectionContainer } from "@ogre-tools/injectable";
+import statusBarItemsInjectable from "./status-bar-items.injectable";
+import type { StatusBarRegistration } from "./status-bar-registration";
+
+describe("<StatusBar />", () => {
+  let render: DiRender;
+  let di: ConfigurableDependencyInjectionContainer;
+
+  beforeEach(async () => {
+    di = getDiForUnitTesting({ doGeneralOverrides: true });
+    render = renderFor(di);
+
+    await di.runSetups();
+  });
+
+  it("renders w/o errors", () => {
+    di.override(statusBarItemsInjectable, () => computed(() => [] as StatusBarRegistration[]));
```

Comment:
By overriding statusBarItems you actually made test less credible. Could we change it back so that it overrides extensions.

Reply:
Okay, would an intermediate dependency be sufficient as well? One whose only job is to `flatMap` from the extensions?

Reply:
There is no need for that. `extensions.get().flatMap(...)` is duplicated yes, but abstracting it would just make things more complicated.

Reply:
Okay, I'll revert.

Reply:
Done